nibiru/utils.py

```diff
@@ -175,3 +178,73 @@ def toPbTimestamp(dt: datetime):
     ts = Timestamp()
     ts.FromDatetime(dt)
     return ts
+
+
+def merge_list(list_of_dict: Iterable[Dict]) -> Dict:
+    """
+    Merge a list of dictionnary together.
+
+    ```
+    >>> d1 = dict(x=0, y=dict(a=0, b=1), z=dict(x=1))
+    >>> d2 = dict(y=dict(c=2), z=dict(y=1))
+    >>> merge_list(d1, d2)
+
+    {
+        "x": 0,
+        "y": {"a": 0, "b": 1, "c": 2},
+        "z": {
+            "x": 1,
+            "y": 1,
+        },
+    }
+    ```
+
+    Args:
+        list_of_dict (Iterable[Dict]): The list of dictionaries to merge
+
+    Returns:
+        Dict: Dictionaries merged together.
+    """
+
+    return reduce(
+        lambda x, y: MERGER.merge(x, y),
+        list_of_dict,
+    )
+
+
+def transform_java_dict_to_list(events: Dict) -> List[Dict]:
+    """
+    Transfrom a java type dictionary to a python dictionary.
+
+    For example:
+
+        {
+            'nibiru.vpool.v1.SwapQuoteForBaseEvent.pair': ['ubtc:unusd'],
+            'tx.fee': ['170unibi'],
+        }
+
+    becomes:
+
+        [
+            {'nibiru': {'vpool': {'v1': {'MarkPriceChanged': {'pair': 'ubtc:unusd'}}}}},
+            {'tx': {'fee': '170unibi'}},
+        ]
```

Comment:
Probably `SwapQuoteForBaseEvent` still stays? Not transformed to `MarkPriceChanged` ?

Reply:
oh yes that was a copy paste mistake